Source/Pawnmorphs/Esoteria/PawnmorpherSettings.cs

```diff
@@ -148,7 +149,9 @@ public override void ExposeData()
 
 			if (Scribe.mode == LoadSaveMode.PostLoadInit)
             {
-                if (formerChance > 1) formerChance /= 100f; 
+                animalBlacklist = null;
```

Comment:
Is this correct?  It seems like it's loading the blacklist and then always discarding it.

Reply:
I already removed it.
It was for debugging default blacklist assignment to check if powerful creatures got set to restricted, and was how i noticed Thrumbo and Elephant was missing.